foursight_core/app_utils.py

```diff
@@ -467,7 +503,111 @@ def view_foursight(self, environ, is_admin=False, domain="", context="/"):
+                access_key_id = credentials.access_key
+                secret_access_key = credentials.secret_key
+                region_name = session.region_name
+                session_token = credentials.token
+                caller_identity = boto3.client("sts").get_caller_identity()
+                user_arn = caller_identity["Arn"]
+                account_number = caller_identity["Account"]
+                aws_credentials_info = {
+                    "AWS Account Number:": account_number,
+                    "AWS User ARN:": user_arn,
+                    "AWS Access Key ID:": access_key_id,
+                    "AWS Secret Access Key:": obfuscate(secret_access_key),
+                    "AWS Region Name:": region_name,
+                    "AWS Session Token:": obfuscate(session_token)
+                }
+                return aws_credentials_info
+            except:
+                return {}
+
+        html_resp = Response('Foursight viewing suite')
+        html_resp.headers = {'Content-Type': 'text/html'}
+        template = self.jin_env.get_template('info.html')
+        env_name = os.environ.get("ENV_NAME")
+        stage_name = self.stage.get_stage()
+        environment_names = {
+            "Environment Name:": env_name,
+            "Environment Name (Full):": full_env_name(env_name),
+            "Environment Name (Short):": short_env_name(env_name),
+            "Environment Name (Inferred):": infer_foursight_from_env(envname=env_name),
+            "Environment Name List:": self.environment.list_environment_names(),
+            "Environment Name List (Unique):": self.environment.list_unique_environment_names()
+        }
+        bucket_names = {
+            "Environment Bucket Name:": self.environment.get_env_bucket_name(),
+            "Foursight Bucket Name:": get_foursight_bucket(envname=env_name, stage=stage_name),
+            "Foursight Bucket Prefix:": get_foursight_bucket_prefix()
+        }
+        gac_name = get_identity_name()
+        gac_values = sorted_dict(obfuscate_dict(get_identity_secrets()))
+        es_host = os.environ.get("ES_HOST")
+        encoded_es_server = gac_values.get("ENCODED_ES_SERVER")
+        environment_and_bucket_info = sorted_dict(obfuscate_dict(self.environment.get_environment_and_bucket_info(env_name, stage_name)))
+        declared_data=sorted_dict(EnvUtils.declared_data())
+        dcicutils_version = pkg_resources.get_distribution('dcicutils').version
+        foursight_core_version = pkg_resources.get_distribution('foursight-core').version
+        versions = {
+            "Foursight-CGAP Version:": version,
+            "Foursight-Core Version:": foursight_core_version,
+            "DCIC-Utils Version:": dcicutils_version,
+            "Python Version:": platform.python_version()
+        }
+        resources = {
+            "Foursight Server:": socket.gethostname(),
+            "Fourfront Server:": environment_and_bucket_info.get("fourfront"),
```

Comment:
Use of `Fourfront` here probably not right, despite the internal identifier matching. I would say `Portal` instead.